ansible_collections/f5networks/f5_modules/plugins/modules/bigip_pool.py

```diff
@@ -702,7 +702,7 @@ class UsableChanges(Changes):
     def monitors(self):
         monitor_string = self._values['monitors']
         if monitor_string is None:
-            return None
+            return ""
```

Comment:
this is not the right way of doing this, if you want to remove monitors from the pool you need to specify the parameter as an empty list, and make sure that in the code we catch this.

So this line needs to be changed accordingly.

One other thing since user provides an empty list in their playbook:

`monitors: []`

You need to modify the property in the Difference class to catch this:

https://github.com/F5Networks/f5-ansible/blob/4e2c1c74b622c6299fbd0f27778690d7b668ac53/ansible_collections/f5networks/f5_modules/plugins/modules/bigip_pool.py#L838

Reply:
The changes are done .